### PR TITLE
feat: add features to text editor

### PR DESCRIPTION
**Pull request by Tj-Tracy**
image

### CODE REVIEW

File: src/components/bulk-email-tool/text-editor/TextEditor.jsx [hunk 1]
```diff
@@ -30,12 +31,13 @@ export default function TextEditor(props) {
         selector: 'textarea#editor',
         height: 600,
         branding: false,
-        plugins: 'advlist code emoticons link lists table',
-        toolbar: 'bold italic | bullist numlist | link emoticons',
+        plugins: 'advlist code emoticons link lists table image language',
+        toolbar: 'formatselect fontselect bold italic underline forecolor | code bullist numlist alignlef aligncenter alignright alignjustify indent | link emoticons image | language',
```
Comment (justinhynes) [inline src/components/bulk-email-tool/text-editor/TextEditor.jsx:35]: A couple of thoughts...

1. In the original editor there is a "dedent"/`decrease indentation` option next to the the indent/"increase indentation" button. Can we add it?

2. In the original editor there is a button labeled `HTML` that allows direct manipulation of the HTML of the message. From a quick glance at the picture, I don't think I see this functionality. Is this still supported?

3. In the original editor there is a "remove link" option. Is that still supported? Should we add it?

4. What is "browser default language"? I don't know if I noticed that before. Should it be added? 

5. In the original editor there is a "blockquote" option. Is this still supported? Should we add it?

**Review by justinhynes — CHANGES_REQUESTED** [review]
Couple of questions before approval

File: src/components/bulk-email-tool/text-editor/TextEditor.jsx (hunk 1 above)
Comment (Tj-Tracy) [reply src/components/bulk-email-tool/text-editor/TextEditor.jsx:35]: tl;dr some buttons no longer exist on the toolbar and are now in context or sub menus due to version differences between here and platform. But, all the functionality is there. I realize the UI changes are not ideal, but I didn't want to spend extra time making custom buttons unless we decide its necessary.

1. This is the one tool I could not find, I don't think it still exists in this version. But, you can just press backspace, so to go back a step (or undo) so I didn't think it was that important.

2. there is an html button in the in the "tools" submenu 
image

3. this is now in a context menu.
image


4. Thats an edx made plugin that an instructor can use to denote a language in an email is not the same as a browser default. Its probably not going to get used, but is a nice-to-have from an i18n standpoint.

5. Block quotes are supposed to be there I just missed it by accident 😅 I've added it
image

**Review by justinhynes — APPROVED** [review]
Thanks for getting this squared away!

**Pull request merged**